### ISSUE TITLE
Placeholder not showing

### ISSUE THREAD

**Issue by zdimaz** [Choices-js/Choices#1145]
new Choices(element, {
    searchEnabled: true,
    removeItemButton: true,
    placeholder: true, // or text !
    allowHTML: true,
    itemSelectText: '', // Press to select
    noChoicesText: 'No choices',
  });
  Select has attr
  <select multiple placeholder="Select one">options</select>
 
 But rendered input not
 <input type="search" name="search_terms" class="choices__input choices__input--cloned" autocomplete="off" autocapitalize="off" spellcheck="false" role="textbox" aria-autocomplete="list" aria-label="null" aria-activedescendant="choices--fui-contactForm-qlxsjf-fields-diensten-item-choice-4" style="min-width: 1ch; width: 9ch;">

**Comment by ahmedtoolapp** [comment]
<select> element does not have placeholder natively 
You have two options : 
- Using <input type="text"> instead of <select>
- Keep <select> and add new empty option to choices/items array

**Comment by Alexbrazdasilva** [comment]
In my case, an option with this definition resolved:

```js
{
    label: "Pick a item",
    placeholder: true,
    disabled: true,
    value: -1, // Or null
  }
  ```

**Comment by Xon** [comment]
> can't you use "placeholderValue" config option?

Sadly there are a bunch of cases where this will be ignored. Plan to fix it in v11.0.0 once I get that branched merged in. For now [v11.0.0RC4](https://github.com/Xon/Choices.js/releases/tag/v11.0.0RC4) is tagged on my fork.
